### DISCUSSION TITLE
Relation with storage folder

### DISCUSSION THREAD

**Discussion by ambroiselebs** [supabase/supabase#17361] [Questions] [answered]
Hello, I'm wondering if there is a way to create a relation key from a table with a folder from my storage.
For exemple I create a folder for each user_id and if I delete the user from the auth panel, the folder from storage is also deleted ?

**Comment by GaryAustin1** [comment, marked as answer]
Folders don't really exist.  They are just a string processing construct from the path with the / separator.  If you delete all the user files with the api then the "folder" will no longer exist.  There is no delete folder api call.  There is also no way to have a foreign key relation to storage.objects and cascade delete.  You have to call the API to do the delete.
